clik/external/hal_cpu/source/cpu_hal.cpp

```diff
@@ -32,12 +35,36 @@
 
 #include "device/device_if.h"
 
+// http://graphics.stanford.edu/~seander/bithacks.html#RoundUpPowerOf2
+static uint64_t round_up_pot(uint64_t v) {
```

Comment:
Given that the link explains that this does not always round up to the next power of two, can you document what this does do, what assumptions it makes? Or is there perhaps an LLVM function whose name we could align with, in which case we could just document it as behaving like the corresponding LLVM function?

Reply:
I've removed this from here, as it's not needed but added a comment in arg_pack.cpp where we still use the same function.